docs/troubleshooting/trouble-shooting.md

```diff
@@ -85,3 +85,23 @@ To improve token security and scalability, the Kubernetes community proposes [KE
 With the GA of the `BoundServiceAccountTokenVolume` feature, the Kubernetes community considers that it is unnecessary to automatically generate tokens for ServiceAccount because it is insecure. Therefore, [KEP-2799](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/2799-reduction-of-secret-based-service-account-token) is proposed. One purpose of this KEP is not to automatically generate token secrets for ServiceAccount, and the other purpose is to clear token secrets generated by unused ServiceAccounts.
 
 For the first purpose, the Kubernetes provides the `LegacyServiceAccountTokenNoAutoGeneration` feature gate, which has entered the Beta phase in Kubernetes v1.24. This is why the Karmada control-plane cannot generate tokens, because `karmada-apiserver` v1.24 is used in Karmada. If you still want to use the previous method to generate a token secret for ServiceAccount, you can refer to this [section](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-an-api-token-for-a-serviceaccount).
+
+## Schedule failed due to "cluster(s) did not have the API resource"
+
+**Karmada detector focus only on resources with karmada-apiserver preferred version.** 
+
+For example, assuming karmada-apiserver is v1.25 version, its HPA resource has both `autoscaling/v1` and `autoscaling/v2` version. 
+However, since the preferred version of HPA is `autoscaling/v2`, the detector will only list/watch `autoscaling/v2` version. 
+If the user creates an HPA of `autoscaling/v1`, kubernetes originally generates create event of both version, but only 
+create event of `autoscaling/v2` been watched by detector.
+
+Basing on this background, you need to pay attention to the following two points:
+
+* **When writing propagation policy, its `resourceSelector` field only supports resource with karmada-apiserver preferred version.**
+* **Member cluster apiserver should support the resource version which karmada-apiserver preferred.**
+
+Put it more specifically, still take HPA as an example, you are advised to use `autoscaling/v2` HPA in both resource template
+and propagation policy. However, if you insist on propagating a `autoscaling/v1` HPA template, you can still succeed if you 
+define `resourceSelector` in propagation policy as `apiVersion: autoscaling/v2`. Then, Karmada finally propagates `autoscaling/v2`
```

Comment:
How about giving this a YAML example?

Reply:
When everyone thinks the English version is OK, I'll write the Chinese version, otherwise it will be a little troublesome to change both sides.